app/models/views/reports/ocmc_export_helper.rb

```diff
@@ -7,7 +7,17 @@ def data
       end
 
       def build_data
-        ActiveRecord::Base.connection.execute(sql_query)
+        office_ids = Rails.application.config.datashare_office_ids
```

Comment:
You don't need this. And if there is a new office you would have to add new office to the list. Just use Office.pluck(:id)